pysrc/models/ind.py

```diff
@@ -11,7 +11,7 @@ class IND:
 
     referenced: list[ColumnInformation]
 
-    errors: list[ErrorMetric] = field(default_factory=list, compare=False)
+    errors: list[MissingValues] = field(default_factory=list, compare=False) #NOTE: THIS IS JUST TEMPORARY SO THAT WE CAN ACTUALLY WORK WITH PARTIAL SPIDER!!!
```

Comment:
I'll try and look into it.

Reply:
@dkuska I think I've found a solution: You can use [`dataclasses-json`](https://pypi.org/project/dataclasses-json/) to parse and write JSON. Just add `@dataclass_json` to each `@dataclass`, and replace `json` calls with the methods from `dataclasses-json`.